README.md

```diff
@@ -228,6 +251,72 @@ ignore (default: `0`)
 - `-latency-p50`: Expected maximum latency for 50th percentile of requests, 0 to
 ignore (default: `0`)
 
+## Observability & Troubleshooting
+
+### What's happening with the rollout of my service?
+
+To view the current state of the rollout of a particular service, you can see
+the service's traffic configuration in [Cloud
+Run](http://console.cloud.google.com/run).
+
+Under the `Revisions` section, the first thing to notice is that the Release
+Manager has automatically assigned the `stable` tag to your current stable
+revision and the `candidate` tag to your last deployed revision. You can also
+see the percentage of traffic each revision is serving.
+
+#### Annotations
+
+For more detailed information on the rollout process, the service object
+contains some annotations with information about the rollout. In Cloud Run, you
+can see the service object in the `YAML` section.
+
+- `rollout.cloud.run/stableRevision` is the name of the current stable revision
+- `rollout.cloud.run/candidateRevision` is the revision name of the current
+  candidate
+- `rollout.cloud.run/lastFailedCandidateRevision` is the last revision that was
+  considered a candidate but failed to meet the health criteria at some point of
+  its rollout process
+- `rollout.cloud.run/lastRollout` contains the last time a rollout occurred
+  (traffic to the candidate was increased)
+- `rollout.cloud.run/lastHealthReport` contains information on why a rollout or
+  rollback occurred. It shows the results of the health assessment and the
+  actual values for each of the metrics
+
+### Errors when rolling out
+
+#### All services
+
+To quickly find out if there are errors when rolling out any of your services,
+you can use the [Logs Viewer](http://console.cloud.google.com/logs). In the
+query builder, add:
+
+```plain
+resource.type = "cloud_run_revision"
+resource.labels.service_name = "release-manager"
+resource.labels.location = "us-central1"
+severity >= ERROR
+```
+
+The above filter will get all the error messages in your Release Manager
+service.
+
+#### A specific service
+
+If you want to filter the errors for a specific service that has opted-in for
+gradual rollout, you can include its service name in the query:
+
+```plain
+resource.type = "cloud_run_revision"
+resource.labels.service_name = "release-manager"
+resource.labels.location = "us-central1"
+jsonPayload.context.data.service = "<YOUR_SERVICE>"
+severity >= ERROR
+```
+
+You can also include a full list of the logs by changing the severity filter to
+`severity >= DEBUG`. You must have set the flag `-verbosity=debug` when
+deploying the Release Manager to have full logs about your rollouts.
```

Comment:
IMO overall this part is not great either.

- the title doesn't reflect "viewing logs of release manager" which is what the section explains

- sub titles are not necessary. you can provide a simple snippet and provide a [Run the query](#) link directly.

- similarly, other filters e.g. severity=, service= can be explained at the end instead of making the section 2x longer

Reply:
Direct links don't work when the `project_id` is removed from the URL parameters. I get
![Screenshot 2020-08-17 at 3 10 42 PM](https://user-images.githubusercontent.com/2955158/90434780-ff89e000-e09b-11ea-9af3-bd4d8c463e8d.png)

Reply:
🤔 is that the case for classic logs viewer as well?
how about you say project='*' or something

Reply:
When I generate the link from the classic logs, it works. It also redirects me to the Preview mode if that was my default mode.